packages/fluent/scss/menu/_layout.scss

```diff
@@ -100,10 +104,11 @@
         }
 
         > .k-separator {
-            margin: 0 var( --kendo-menu-separator-margin, #{$kendo-menu-separator-margin} );
+            margin-block: var(--kendo-menu-separator-margin-block, #{$kendo-menu-separator-margin-block});
+            margin-inline: var(--kendo-menu-separator-margin-inline, #{$kendo-menu-separator-margin-inline});
             width: 0;
             height: auto;
-            border-bottom-width: var( --kendo-menu-separator-width, $kendo-menu-separator-width);
+            border-left-width: var( --kendo-menu-separator-width, $kendo-menu-separator-width);
```

Comment:
The syntax here should be: border-left-width: var( --kendo-menu-separator-width, #{$kendo-menu-separator-width});